style.css

```diff
@@ -1,4 +1,74 @@
 body {
-background-color: #232323
+  color: #ffffff;
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 15px;
+  width: auto;
+  background: url(nieckovsky_image_main.jpg);
+  background-color: #111415;
+  background-size: 100%;
+  background-position: center;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+}
+
+header {
+  width: 100%;
+  height: 50px;
+}
+
+nav {
+  display: inline-block;
+  width: 100%;
+  position: relative;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+span {
+  display: inline-block;
+  width: 125px;
+  height: 50px;
+  background-image: url(nieckovsky_logo_top.png);
+  background-size: 125px 50px;
+  background-repeat: no-repeat;
+  font-size: 0;
+  line-height: 0;
+  vertical-align: middle;
+  position: absolute;
+  margin-top: auto;
+  margin-bottom: auto;
+  margin-left: 5%;
+}
+
+ul {
+  list-style-type: none;
+  vertical-align: middle;
+  height: 26px;
+  margin-left: 40%;
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+li {
+  display: inline-block;
+  vertical-align: text-top;
+  padding: 5px 20px 5px 20px;
+}
+
+a {
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 300;
+  color: inherit;
+  padding: 0 30px;
+}
 
+footer {
+  color: rgba(255, 255, 255, 0.15);
+  width: auto;
+  margin: 8vh auto;
+  position: absolute;
```

Comment:
Okay, I get it that you would like your footer on the bottom of the page.
And your solution is fine for that.

However, what usually is pushing your footer to the bottom is well.. content. You can use some lorem ipsum for that.

What's more important is that using footer with position absolute may lead to worse user experience. Let's say you have some blogposts on your page. These are usually long, so you have scroll bar on the right. Usually you would like to read whole post and then see footer. However, using position absolute makes sure that you're footer will always be on the bottom of your browser. So, with scroll bar, it will be under your lorem ipsum. Always. Makes up for bad UX, doesn't it?

Reply:
I thought that would be a sufficient solution for this particular project since I'm not planning on adding any lengthy blog posts, but I see your point. Nonetheless, I'd like for the footer to stay in one place on every subpage, so can I keep it that way? Or should I just mess with my content dimensions separately for it to stay where I want it to be (so there's no "jumping up and down" between the pages)? I also have considered adding a background to the footer as well as the navigation bar and pushing them a little bit higher with z-index, so together (just in case) they would work as a neat frame for my overflowing content.

Reply:
We should avoid any jumping we can. I don't really know what you mean by frame (should it cover the content in the bottom?)  As long as UX is okay and there's no jumping I can get behind your solution :)

Reply:
Yes, I meant just making the navbar and footer placed at the top and bottom of the window; also: forgot to mention - of course with position fixed, so they stay put, and the content flows underneath.
Good to know I figured it out good enough. :)